model/infinispan/pom.xml

```diff
@@ -75,12 +75,30 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
```

Comment:
In Keycloak, the preference was to not use mockito until now. At the same time, I can see it useful in some cases like this. So I hope that this change is accepted, but will discuss with the wider Keycloak team and will let you know adding mockito is an issue.